### ISSUE TITLE
[EPIC] SLO Definition

### ISSUE THREAD

**Issue by chopchop505** [chopchop505/surfana#19]
## TLDR
This epic will ensure your service has well defined SLOs from the perspective of the user.  As part of the review, we will be looking at your reasoning for definining the SLO, how you monitor, and continuous re-evaluate your SLOs

## Resources
[SLOs at Shopify](https://development.shopify.io/engineering/production/overview/service_level_objectives)
[Google SRE](https://sre.google/sre-book/service-level-objectives/)
[Defining SLOs in Datadog]
## Shopify Examples
[Shopstatus SLOs]()## Maturity Levels
1. **UNMANAGED** No SLO or SLO does not represent user requirements.
2. **MANAGED** Service has defined SLOs. Service has measurement of SLO but it is ad hoc or undocumented.
3. **DEFINED** Service has defined SLOs. Measurement is a documented process with clear owner
4. **MEASURED** SLO is published to users. SLO is measured automatically. SLO represents user requirements.
5. **CONTINUOUS IMPROVEMENT** SLO is published to users. SLO is measured automatically. SLO represents user requirements.

## Exit Criteria
**Complete the following sections in sections in [PRR Document]()**
- [ ] Critical User Journeys
- [ ] SLI Definition
- [ ] SLO Target Goals and SLO Measurment Period

**Complete the following in Datadog**
- [ ] Create SLI, SLO, and Error Dashboard Dashboard
- [ ] Create SLO Monitors with Pageable alerts

**Complete the following in ServicesDB**
- [ ] Verify SLOS are imported and visible on your service page
